samples/unit-tests/stock-tools/stock-annotation-dragging/demo.js

```diff
@@ -55,6 +55,15 @@ QUnit.test('Dragging annotation', assert => {
                 dataGrouping: {
                     enabled: true
                 }
+            }, {
+                type: 'flags',
+                data: [{
+                    x: data[50][0]
+                }, {
+                    x: data[52][0]
+                }, {
+                    x: data[55][0]
+                }]
```

Comment:
Please add this series after existing tests. We don't know if adding this series prior to the test won't make the test redundant (and not testing what it should test).